torch/csrc/lazy/core/helpers.cpp

```diff
@@ -10,6 +11,7 @@ std::vector<int64_t> DropDimensions(
     c10::ArrayRef<int64_t> sizes,
     c10::ArrayRef<int64_t> drop_dims) {
   std::vector<int64_t> new_dims;
+  new_dims.reserve(std::max(sizes.size() - drop_dims.size(), 0));
```

Comment:
std::max is unnecessary because if `sizes.size() < drop_dims.size()` there is unsigned int underflow..